drivers/SmartThings/matter-switch/src/utils/switch_utils.lua

```diff
@@ -63,6 +63,14 @@ function utils.mired_to_kelvin(value, minOrMax)
   end
 end
 
+function utils.check_vendor_overrides(device, override_key)
+  for product_id, overrides in pairs(fields.vendor_overrides[device.manufacturer_info.vendor_id] or {}) do
+    if product_id == device.manufacturer_info.product_id and overrides[override_key] ~= nil then
+      return true
```

Comment:
```suggestion
    if product_id == device.manufacturer_info.product_id then
      return overrides[override_key]
```
One suggestion I was thinking of would be return this so it can be checked/used at the callsite